segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/OperationProcessor.java

```diff
@@ -295,13 +304,19 @@ private int getFetchCount() {
     }
 
     /**
-     * Determines if the {@link #operationQueue} has any {@link CompletableOperation} with an {@link OperationPriority}
-     * that requires an immediate execution (i.e., {@link OperationPriority#isThrottlingExempt()} is true).
+     * Determines whether we have to notify the {@link Throttler} to suspend current delay. This can happen in two cases:
+     * i) If the {@link #operationQueue} has been closed (e.g., Bookkeeper error) while experiencing a maximum delay;
+     * ii) If the {@link #operationQueue} has any {@link CompletableOperation} with an {@link OperationPriority} that
+     * requires an immediate execution (i.e., {@link OperationPriority#isThrottlingExempt()} is true).
      *
      * @return True if throttling needs to be suspended, false otherwise.
      */
     @VisibleForTesting
-    protected boolean hasThrottleExemptOperations() {
+    protected boolean hasToSuspendThrottlingDelay() {
```

Comment:
For clarity lets avoid double negatives. Currently this is method that returns true if something should not happen. Instead lets reverse the return value and rename the method accordingly.

Reply:
I would prefer not to do this, because the current (and previous) naming of this method in `OperationProcessor` is in line with what `Throttler` class expects. The `Throttler` class, which uses this method as a callback, expects it to work as:
```
@param isSuspended A Supplier that will be invoked upon every call to {@link #throttle()}. If it returns true,
     *                    no throttling will be applied; if it returns false, normal throttling logic is executed.
 ```
That is, the naming (and logic) of this method is aligned with what the user of that function expects, so I would prefer to leave things in its current form, as the act itself of "suspending throttling" has a relevant meaning on its own.